git_machete/cli.py

```diff
@@ -793,6 +793,9 @@ def strip_remote_name(remote_branch: RemoteBranchShortName) -> LocalBranchShortN
     except MacheteException as e:
         sys.stderr.write(f"\n{e}\n")
         exit_script(1)
+    except UnprocessableEntityHTTPError as e:
```

Comment:
Maybe just make `UnprocessableEntityHTTPError` a subclass of `MacheteException`? I suppose then the former will be captured by `except MacheteException` clause